docs/loading/RoutineLoad.md

```diff
@@ -268,6 +268,128 @@ After submitting the load job, you can execute the [SHOW ROUTINE LOAD](../sql-re
     >
     > You do not need to specify the `COLUMNS` parameter if the names and number of the keys in the JSON object completely match those of fields in the StarRocks table.
 
+### Load Avro-format data
+
+Since v3.0.1, StarRocks supports loading Avro data by using Routine Load.
+
+#### Prepare a dataset
+
+**Avro schema**
+
+1. Create the following  Avro schema file `avro_schema.avsc`:
+
+      ```JSON
+      {
+          "type": "record",
+          "name": "sensor_log",
+          "fields" : [
+              {"name": "id", "type": "long"},
+              {"name": "name", "type": "string"},
+              {"name": "checked", "type" : "boolean"},
+              {"name": "data", "type": "double"},
+              {"name": "sensor_type", "type": {"type": "enum", "name": "sensor_type_enum", "symbols" : ["TEMPERATURE", "HUMIDITY", "AIR-PRESSURE"]}}  
+          ]
+      }
+      ```
+
+2. Register the Avro schema in the [Schema Registry](https://docs.confluent.io/platform/current/schema-registry/index.html).
+
+**Avro data**
+
+Prepare the Avro data and send it to the Kafka topic `topic_0`.
+
+#### Create a table
+
+According to the fields of Avro data, create a table `sensor_log` in the target database `sensor` in the StarRocks cluster. The column names of the table must match the field names in the Avro data. For the mapping of data types between the table columns and the Avro data fields, see [Data types mapping](#Data types mapping).
+
+```SQL
+CREATE TABLE sensor.sensor_log ( 
+    `id` bigint NOT NULL COMMENT "sensor id",
+    `name` varchar(26) NOT NULL COMMENT "sensor name", 
+    `checked` boolean NOT NULL COMMENT "checked", 
+    `data` double NULL COMMENT "sensor data", 
+    `sensor_type` varchar(26) NOT NULL COMMENT "sensor type"
+) 
+ENGINE=OLAP 
+PRIMARY KEY (id) 
+DISTRIBUTED BY HASH(`id`) BUCKETS 5; 
+```
+
+#### Submit a Routine Load job
+
+Execute the following statement to submit a Routine Load job named `sensor_log_load_job` to consume the Avro messages in the Kafka topic `topic_0` and load the data into the table `sensor_log` in the database `sensor`. The load job consumes the messages from the initial offset in the specified partitions of the topic.
+
+```SQL
+CREATE ROUTINE LOAD example_db.sensor_log_load_job ON sensor_log1  
+PROPERTIES  
+(  
+    "format" ="avro"  
+)  
+FROM KAFKA  
+(  
+    "kafka_broker_list" ="<kafka_broker1_ip>:<kafka_broker1_port>,<kafka_broker2_ip>:<kafka_broker2_port>,...",
+    "confluent.schema.registry.url"="http://172.xx.xxx.xxx:8081",  
+    "kafka_topic"="topic_0",  
+    "kafka_partitions"="0,1,2,3,4,5",  
+    "property.kafka_default_offsets"="OFFSET_BEGINNING"  
+);
+```
+
+- Data Format
+
+  You need to specify `"format"="avro"` in the clause `PROPERTIES` to define that the data format is Avro.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/cfdc090f-d61a-4b2d-a15e-96be5a2dbe70)
If you add spaces preceding and following equation marks (=) in code snippets, do not forget to add spaces here.